curriculum/challenges/chinese/01-responsive-web-design/basic-html-and-html5/add-placeholder-text-to-a-text-field.chinese.md

```diff
@@ -24,12 +24,14 @@ localeTitle: 给输入框添加占位符文本
 
 ```yml
 tests:
-  - text: '给现有的<code>input</code>输入框添加一个<code>placeholder</code>属性。'
+  - text: 给现有的<code>input</code>输入框添加一个<code>placeholder</code>属性。
     testString: assert($("input[placeholder]").length > 0);
-  - text: '设置<code>placeholder</code>属性的值为 ”猫咪图片地址“。'
+  - text: 设置<code>placeholder</code>属性的值为 ”猫咪图片地址“。
```

Comment:
Same kind of issue as I mentioned in the `Use the u Tag to Underline Text` above. `"cat photo URL"` needs to be wrapped in `code` tags on the English version, because this should not be translated.  Leave it for now.